Documentation~/images/CinemachineConfiner2Dc.md

```diff
@@ -0,0 +1,35 @@
+# Cinemachine Confiner 2D
+
+![](images/CinemachineConfiner2Dc.png)
+
+Use the Cinemachine Confiner 2D [extension](https://docs.unity3d.com/Packages/com.unity.cinemachine@2.6/manual/CinemachineVirtualCameraExtensions.html) to confine the camera’s position so that the screen edges stay within a shape defined by a 2D polygon. This works for orthographic or perspective cameras, provided that the camera's forward vector remains parallel to the bounding shape’s normal (that is, that the camera is looking straight at the polygon, and not obliquely at it).
+
+When confining the camera, the Cinemachine Confiner 2D considers the camera’s view size at the polygon plane, and its aspect ratio. Based on this information and the input polygon, it computes a second (smaller) polygon, and constrains the camera’s transform to it. Computation of this secondary polygon is resource-intensive, so you should only do this when absolutely necessary. 
+
+Necessary use cases in which you need to recompute the cached secondary polygon include:
+
+- when the input polygon’s points change,
+- when the input polygon is non-uniformly scaled.
+
+In these cases, for efficiency reasons, Cinemachine does not automatically regenerate the inner polygon. The client needs to call the InvalidateCache() method to trigger the recalculation. You can do this from; 
+
+- the script by calling InvalidateCache, or 
+- the component inspector; to do so, press the **Invalidate Cache** button.
+
+If the input polygon scales uniformly or translates or rotates, the cache remains valid. 
+
+## Oversize Windows
+Oversize WindowIf sections of the confining polygon are too small to fully contain the camera window, Cinemachine calculates a polygon skeleton for those regions. This is a shape with no area, that serves as a place to put the camera when it is confined to this region of the shape.
```

Comment:
Space missing before If

Reply:
Hmmm... this file isn't to be used. Where did you find it?